jupyterlab_bigquery/jupyterlab_bigquery/query_incell_editor.py

```diff
@@ -0,0 +1,16 @@
+from ipywidgets import DOMWidget
+from traitlets import Unicode
+
+# TODO: refactor name and version to sync with front 
+module_name = 'bigQuery-query-incell-editor'
```

Comment:
Let's not mix the camelCase and kebal-case. snake_case would be preferred. Can this be rename to `bigquery_query_incell_editor`?

Same thing for the other locations.

Reply:
I fixed the module_name.  The other names in `Unicode` need to correspond to class names in front-end, so I did not change them.  Should I?

Reply:
It would be nice if all the names are consistent. Preferably change `bigQuery-query-incell-editor` to `bigquery_query_incell_editor` everywhere you can.